django/test/runner.py

```diff
@@ -601,6 +601,8 @@ def setup_test_environment(self, **kwargs):
     def load_tests_for_label(self, label, discover_kwargs):
         label_as_path = os.path.abspath(label)
         tests = None
+        default_test_name_patterns = self.test_loader.testNamePatterns
```

Comment:
Nit: I would call this "saved" or "original" patterns, because technically it might not be the default.